lib/container_server_linux.go

```diff
@@ -9,12 +9,16 @@ import (
 )
 
 func (c *ContainerServer) addSandboxPlatform(sb *sandbox.Sandbox) {
-	c.state.processLevels[selinux.NewContext(sb.ProcessLabel())["level"]]++
+	// NewContext() always returns nil, so we can safely ignore the error
+	ctx, _ := selinux.NewContext(sb.ProcessLabel())
```

Comment:
I would still go for the error handling here since I have the feeling that it will be missed when the day comes that `selinux.NewContext` returns a real error.

Reply:
Agreed, from a maintainability perspective we should catch if there ever becomes an error

Reply:
I think that @rhatdan wants to get rid of those entirely by letting containers/storage handle all label issue but I can update the code now to be on the safe side.

Reply:
I've had a look at the call stack and don't think it's worth the effort. We had to change a whole bunch of APIs up to the ContainerServer which does not really implement error handling (even for {Add,Remove}Container).

I agree that errors shouldn't be ignored but given this affects large parts of the API, I prefer to defer this to a separate PR.